frontend/src/features/comparisons/Comparison.tsx

```diff
@@ -240,6 +241,20 @@ const Comparison = ({ afterSubmitCallback }: Props) => {
           otherUid={uidA}
         />
       </Grid>
+      <Grid
+        item
+        xs={12}
+        sx={{
+          mt: 1,
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: 'column',
```

Comment:
we may want to add style rule to avoid taking up space here on the video comparison page, when the helper is empty:
```suggestion
          flexDirection: 'column',
          '&:empty': {
            display: 'none',
          },          
```